package.json

```diff
@@ -40,32 +45,47 @@
   "homepage": "https://github.com/Zooz/api-schema-validator#readme",
   "dependencies": {
     "api-schema-builder": "^1.1.0",
+    "jest-diff": "^24.8.0",
+    "jest-matcher-utils": "^24.8.0",
     "chalk": "^2.4.2",
     "columnify": "^1.5.4",
     "lodash": "^4.17.11"
   },
   "devDependencies": {
+    "@types/jest": "^24.0.15",
```

Comment:
There is an unfortunate consequence of this being a devDependency - TS compilation will fail for people who use TS but don't use Jest, unless they include Jest typings themselves. Alternative is to move it to prodDependencies, but then people will transiently receive types in runtime as well. I'm not sure how you would want to proceed here.